src/speechless/processing/analysis/spectrogram.py

```diff
@@ -76,22 +76,15 @@ def calc_indicator(sig: np.ndarray, stream_info: Dict[StreamInfo, object]) -> np
     # it generates one more for some reason
     y = librosa.power_to_db(
         librosa.feature.melspectrogram(sig, sr=SAMPLE_RATE, n_fft=N_FFT, hop_length=SEG_LEN).T)
-    y -= np.median(y)
+    y -= np.mean(y)
```

Comment:
Maybe let's also consider something like this:
```suggestion
    y -= np.min(y) # move to the positive values
    y -= 0.2*np.max(y) 
```
This way, the total length of silent fragments in the recording won't affect this process

Reply:
We could describe this problem and provide this as a possible solution in the document.